### PR TITLE
Feature/stage color

### PR DESCRIPTION
**Pull request by xihai01**
Add an api route called api/map_color which queries the health regions table and sends a JSON object to the client.
In the client side, each health region can now change color depending on which stage it is.
Classnames package is installed to handle stage colors.
Data fetching code refactored to use promise.all().
Seeds file refactored to update PHU ID's.
Rspec gem installed for testing.
Rspec test created for health region model.

### CODE REVIEW

**Comment by xihai01** [comment]
@aalibarre @simplyDonald

File: server/config/routes.rb [hunk 1]
```diff
@@ -14,9 +14,10 @@
   # See how all your routes lay out with "rake routes".
 
   namespace :api do
-
-    resources :maps, only: [:index]
-
+    resources :users
```
Comment (simplyDonald) [inline server/config/routes.rb]: do we need a users route?

File: server/app/assets/javascripts/api/map_color.coffee [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (simplyDonald) [inline server/app/assets/javascripts/api/map_color.coffee]: will you be using this file?

File: server/app/assets/stylesheets/api/map_color.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the api/map_color controller here.
```
Comment (simplyDonald) [inline server/app/assets/stylesheets/api/map_color.scss]: dormant file?

File: server/app/helpers/api/map_color_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module Api::MapColorHelper
```
Comment (simplyDonald) [inline server/app/helpers/api/map_color_helper.rb]: dormant file?

File: client/src/hooks/useApplicationData.js [hunk 5]
```diff
@@ -1,40 +1,35 @@
...
-    // get geoJSON from server
-    d3.json("http://localhost:3000/api/maps")
-      .then((data) => {
-        console.log(data);
+    // load geoJSON data and stage # data for each region
+    Promise.all([
+      d3.json("http://localhost:3000/api/maps"),
+      axios.get("http://localhost:3000/api/map_color"),
+    ])
+      .then((response) => {
```
Comment (simplyDonald) [inline client/src/hooks/useApplicationData.js]: you might prefer array-destructuring syntax for Promise.all on line 21:
.then(([mapData,heaalthData])=>{})

File: server/config/routes.rb (hunk 1 above)
Comment (simplyDonald) [inline server/config/routes.rb]: /api/user route has been redundant & was removed in a previous commit

File: server/config/routes.rb [hunk 6]
```diff
@@ -14,9 +14,10 @@
...
+    resources :maps, only: [:index] # fetches geoJSON
+    resources :map_color, only: [:index] # fetches stage colors of regions
+    resources :admin
```
Comment (simplyDonald) [inline server/config/routes.rb]: confirm that admin route has been moved into its own namespace in your most recent  commit

**Review by simplyDonald — CHANGES_REQUESTED** [review]

**Review by xihai01 — COMMENTED** [review]
did array restrucuring and removed empty files.
removed users and admin routes too.

**Review by simplyDonald — APPROVED** [review]
looks good

**Review by simplyDonald — APPROVED** [review]
user and admin route changes resolved

**Review by simplyDonald — APPROVED** [review]
looks good

**Review by simplyDonald — APPROVED** [review]

**Pull request merged**